util/src/hex.rs

```diff
@@ -61,3 +68,24 @@ pub fn public_sign_key_from_hex_string(hex: &str) -> Result<PublicSigningKey, St
     PublicSigningKey::from_bytes(&bytes)
         .map_err(|e| format!("error deserializing signing key from bytes: {e}"))
 }
+
+/// Convert a Baby-JubJub point to a hex string
+pub fn jubjub_to_hex_string(point: &BabyJubJubPoint) -> String {
+    let converted_point = Projective::<EmbeddedCurveConfig>::from(*point);
+    let mut bytes = vec![];
+    converted_point.into_affine().serialize_uncompressed(&mut bytes).unwrap();
+
+    format!("0x{}", hex::encode(bytes))
+}
+
+/// Deserialize a Baby-JubJub point from a hex string
+pub fn jubjub_from_hex_string(hex: &str) -> Result<BabyJubJubPoint, String> {
+    // Deserialize as a string and remove "0x" if present
+    let stripped = hex.strip_prefix("0x").unwrap_or(hex);
+    let bytes = hex::decode(stripped)
+        .map_err(|e| format!("error deserializing bytes from hex string: {e}"))?;
+
+    let projective = Projective::<EmbeddedCurveConfig>::deserialize_uncompressed(bytes.as_slice())
+        .map_err(raw_err_str!("error deserializing projective point from bytes: {:?}"))?;
+    Ok(projective.into())
+}
```

Comment:
neat, I'll use these in the contract deploy scripts as well to parse a protocol pubkey from the CLI for initialization